prerendering.bs

```diff
@@ -658,6 +673,48 @@ The <dfn attribute for="PerformanceNavigationTiming">activationStart</dfn> gette
 
 1. Return a {{DOMHighResTimeStamp}} with a time value equal to the [=current document=]'s [=Document/activation start time=].
 
+<h3 id="client hint cache">Interaction with Client Hint Cache</h3>
+
+We need to ensure that the [=Accept-CH cache=], which is owned by user agent as a global storage, is not modified while prerendering, but is properly updated after activation. The following modifications ensure this.
+
+Each [=prerendering browsing context=] has a <dfn for="prerendering browsing context">prerender-scoped Accept-CH cache</dfn>, which is an [=ordered map=] from [=origin=] to [=client hints set=].
+
+This stores which client hints each origin has opted into receiving, until it can be copied to the global [=Accept-CH cache=] when [=prerendering browsing context/finalize activation|activation is finalized=].
+
+The [=prerendering browsing context/prerender-scoped Accept-CH cache=] is an [=ordered map=] of [=origins=] to [=client hints sets=].
```

Comment:
This seems redundant with line 680, but it uses the preferred phrasing ("of origins to client hint sets"). So let's change line 380 to this style and delete this line.

Reply:
Thanks! I updated that line and removed this line.